uber/site_sections/devtools.py

```diff
@@ -1,8 +1,10 @@
 from uber.common import *
```

Comment:
This whole file can probably go away, but I'm leaving it around until I confirm that for sure.